### PR TITLE
tools: install luks-encrypt-storage script by guest-components

### PR DESCRIPTION
**Pull request by ChengyuZhu6**
Install luks-encrypt-storage script by guest-components. So that we can maintain a single source and prevent synchronization issues.

Fixes: #10173 -- part I

### CODE REVIEW

**Comment by ChengyuZhu6** [comment]
cc @wainersm @fitzthum

**Review by fidencio — APPROVED** [review]
lgtm, thanks @ChengyuZhu6!

**Review by GabyCT — APPROVED** [review]

File: tools/osbuilder/rootfs-builder/rootfs.sh [hunk 1]
```diff
@@ -750,11 +750,6 @@ EOF
 		tar xvJpf ${COCO_GUEST_COMPONENTS_TARBALL} -C ${ROOTFS_DIR}
 	fi
 
-	if [ "${MEASURED_ROOTFS}" == "yes" ]; then
```
Comment (wainersm) [inline tools/osbuilder/rootfs-builder/rootfs.sh:753]: yeah, I didn't mention on the other PR but I wasn't so comfortable with this checking. Measure rootfs is a feature that we likely need to revisit. I'm glad that you changed it ;)

**Comment by wainersm** [comment]
The job on TDX was cancelled so I re-triggered to see if the tests continue to pass there (and they should!)

**Review by wainersm — APPROVED** [review]
hey @ChengyuZhu6 ! Thanks!

**Pull request merged**